### PR TITLE
Fix PMD warning

### PR DESCRIPTION
**Pull request by Taeir**
Changed test method so that PMD is happy

<bug, testing

### CODE REVIEW

File: src/test/java/com/github/fishio/logging/TestTxtFileHandler.java [hunk 1]
```diff
@@ -76,9 +76,15 @@ public void testSetGetBW() {
...
 	 * Test if output calls correct method.
+	 * 
+	 * @throws IOException
+	 * 		we verify that write was called on our mocked buffered writer,
+	 * 		but the exception has to be caught. Since we are only verifying
+	 * 		and not actually calling the method, this exception cannot occur.
+	 * 		This throws is here to make the code cleaner.
 	 */
 	@Test
-	public void testOutput() {
```
Comment (Diocruel) [inline src/test/java/com/github/fishio/logging/TestTxtFileHandler.java:81]: _Auto-generated comment_

```
JUnit tests should include assert() or fail()
```

Seems that PMD is still not that happy!

< layout

Comment (Taeir) [inline src/test/java/com/github/fishio/logging/TestTxtFileHandler.java:81]: @Diocruel That will be fixed in PR #217, commit 0ee147c. This PR still makes PMD happier, by removing one warning a few lines below this one.

<layout

**Comment by Diocruel** [comment]
Okay looks good to merge then!

<layout

**Comment by ChielBruin** [comment]
Looks good, I will merge it

<testing

**Pull request merged**